src/stores/configureStore.js

```diff
@@ -1,23 +1,18 @@
 import { applyMiddleware, compose, createStore } from 'redux';
 import { routerMiddleware } from 'connected-react-router';
-import history from './history';
```

Comment:
Because I needed to access the store also in the giantswarm client wrapper, I made a weird singleton out of history.

I found a better way to solve that now.